packages/foam-core/src/utils/hashtags.ts

```diff
@@ -1,6 +1,6 @@
 import { isSome } from './core';
-const HASHTAG_REGEX = /(^|\s)#([0-9]*[\p{L}_-][\p{L}\p{N}_-]*)/gmu;
-const WORD_REGEX = /(^|\s)([0-9]*[\p{L}_-][\p{L}\p{N}_-]*)/gmu;
+const HASHTAG_REGEX = /(^|\s)#([0-9]*[\p{L}\/_-][\p{L}\p{N}\/_-]*)/gmu;
```

Comment:
can you please add a linter ignore directive in this and the following line?
we want to keep linter warnings to zero

Reply:
Turns out the escaping isn't necessary.